deepcell_tracking/utils.py

```diff
@@ -236,6 +302,10 @@ def save_trks(filename, lineages, raw, tracked):
             trks.add(tracked_file.name, 'tracked.npy')
             os.remove(tracked_file.name)
 
+    if isinstance(filename, io.BytesIO):
+        filename.seek(0)
+        return filename
```

Comment:
I'm not a fan of the conditional returns.

IMO the `save_X` functions don't need to return anything, as they are just supposed to save data.

Reply:
I had assumed that we were modifying BytesIO and therefore needed to return the modified object to the user. Am I misunderstanding how BytesIO is working?

Reply:
I'm not 100% sure myself but I think that because you are passing in the BytesIO object, the function is modifying it. Can you verify whether or not this is the case?